rest-api-spec/src/main/resources/rest-api-spec/api/text_structure.find_field_structure.json

```diff
@@ -0,0 +1,103 @@
+{
+  "text_structure.find_field_structure":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/find-field-structure.html",
+      "description":"Finds the structure of a text field in an index."
+    },
+    "stability":"stable",
+    "visibility":"public",
+    "headers":{
+      "accept": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_text_structure/find_field_structure",
+          "methods":["GET"]
+        }
+      ]
+    },
+    "params":{
+      "index":{
+        "type":"string",
+        "description":"The index containing the analyzed field",
+        "required":true
+      },
+      "field":{
+        "type":"string",
+        "description":"The field that should be analyzed",
+        "required":true
+      },
+      "lines_to_sample":{
+        "type":"int",
+        "description":"How many documents should be included in the analysis",
+        "default":1000
+      },
+      "line_merge_size_limit":{
+        "type":"int",
+        "description":"Maximum number of characters permitted in a single message when lines are merged to create messages.",
+        "default":10000
+      },
```

Comment:
This parameter shouldn't be needed for this endpoint, as it should assume the field is the entire message, not just one line of it.